digits/templates/models/images/classification/show.html

```diff
@@ -76,6 +76,7 @@ <h4 class='text-center'>Dataset</h4>
 
             {% set task = job.train_task() %}
             <hr>
+            <h2>Test Model</h2>
```

Comment:
I am unsure about this but perhaps the 'Test Model' title should be moved down to after the 'Download Model' button?